scripts/get-butter.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+
+const request = require('sync-request');
+const res = request('GET', 
+    'https://api.buttercms.com/v2/posts/?page=1&page_size=1000000&auth_token=fca1bc914c05371263f98c9d3480eaa215fd63d6', 
```

Comment:
Is there a possibility to hide the auth token since this repo is public?

Reply:
Hm, yes we could. The token will end up in the client though anyway (it's considered a client-safe token, plus the API is read-only, so basically no security risk), but I agree this is a good practice. 

Where's a good place to move it? Not sure if Stencil has an environment capability like Angular?

Reply:
We can use something like in this post https://medium.com/stencil-tricks/environment-variables-with-stenciljs-57e9da591280

Reply:
This is now resolved with the merged branch from Sam (butter-api-key)